shared/fs/common/path-item-action/layout.js

```diff
@@ -52,7 +52,7 @@ const getRawLayout = (path: Types.Path, pathItem: Types.PathItem): Layout => {
         ...empty,
         copyPath: true,
         ignoreTlf: true,
-        sendLinkToChat: false, // TODO enable mobile // desktop uses separate button
+        sendLinkToChat: isMobile, // desktop uses separate button
```

Comment:
This probably shouldn't show when it leads to the 'Copy link' screen, since `copyPath` does that with one fewer step.